app/components/wallet/summary/WalletSummary.js

```diff
@@ -59,16 +59,20 @@ export default class WalletSummary extends Component {
             {amount}
             <img src={adaSymbolBig} role="presentation" />
           </div>
-          <div className={styles.pendingConfirmation}>
-            {`${intl.formatMessage(messages.pendingIncomingConfirmationLabel)}`}
-            : {pendingAmount.incoming.toFormat(DECIMAL_PLACES_IN_ADA)}
-            <img src={adaSymbolSmallest} role="presentation" />
-          </div>
-          <div className={styles.pendingConfirmation}>
-            {`${intl.formatMessage(messages.pendingOutgoingConfirmationLabel)}`}
-            : {pendingAmount.outgoing.toFormat(DECIMAL_PLACES_IN_ADA)}
-            <img src={adaSymbolSmallest} role="presentation" />
-          </div>
+          {pendingAmount.incoming.toFormat() > 0 &&
```

Comment:
That won't work correctly (or is not wise) because [toFormat](http://mikemcl.github.io/bignumber.js/#toFor) returns a string. I will update it to `greaterThan(0)` which is the appropriate method in this case 😉